azext_edge/tests/edge/support/create_bundle/test_orc_int.py

```diff
@@ -0,0 +1,62 @@
+# coding=utf-8
+# ----------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License file in the project root for license information.
+# ----------------------------------------------------------------------------------------------
+
+from knack.log import get_logger
+from azext_edge.edge.common import OpsServiceType
+from .helpers import check_non_custom_file_objs, get_file_map, run_bundle_command
+
+logger = get_logger(__name__)
+
+
+def test_create_bundle_orc(init_setup, tracked_files):
+    """Test for ensuring file names and content. ONLY CHECKS orc."""
+    ops_service = OpsServiceType.orc.value
+    command = f"az iot ops support create-bundle --ops-service {ops_service}"
+    walk_result = run_bundle_command(command=command, tracked_files=tracked_files)
+    file_map = get_file_map(walk_result, ops_service)
+
+    # TODO: add in expected for each
+    # orc
+    # do we always have these? What cases do they have them vs not?
```

Comment:
This is what a vanilla `init` deployment gets you, which can change as soon as you start using the system.

Reply:
yes - first iteration is trying to make sure that in "any" normal state the tests wont break and then will try adding more specific things ( will make a "living doc" of what states we want to check)